lib/kernel.ts

```diff
@@ -1009,14 +1026,14 @@ export class Kernel {
 				payload.schema,
 				payload.options?.mask,
 			);
-			const cards = await stream.query(
+			const contracts = await stream.query(
 				query.selected,
 				query.filteredQuery,
 				payload.options,
 			);
 			stream.emit('dataset', {
 				id: payload.id,
-				cards,
+				contracts,
```

Comment:
This is also a breaking change. `cards: contracts` works.

Reply:
It's a "subtle" point :)

Reply:
There was indeed another similar breaking change in a test file.

Reply:
Yeah.... renaming stuff can be complicated :p